corehq/ex-submodules/couchexport/export.py

```diff
@@ -309,7 +309,8 @@ def split_path(path):
         id = []
         for k in path:
             if isinstance(k, basestring):
-                column.append(k)
+                if k:
+                    column.append(k)
```

Comment:
I don't have a huge amount of confidence that this won't break something else though I can't think of anything right now.